grails-app/views/project/externalCSProjectTemplate.gsp

```diff
@@ -105,22 +105,8 @@
         var viewModel = new ViewModel();
         ko.applyBindings(viewModel);
 
-        if (viewModel.mainImageUrl()) {
-            $( '#carousel' ).sliderPro({
-                width: '100%',
-                height: 'auto',
-                autoHeight: true,
-                arrows: false, // at the moment we only support 1 image
-                buttons: false,
-                waitForLayers: true,
-                fade: true,
-                autoplay: false,
-                autoScaleLayers: false,
-                touchSwipe:false // at the moment we only support 1 image
-            });
-        }
 
-        initialiseProjectArea();
+        //initialiseProjectArea();
```

Comment:
Given that we don't use sliderpro anymore, I think its safe to remove the js library,
Pls remove commented line.